tests/test_quality.py

```diff
@@ -29,7 +29,7 @@ class TestQuality(unittest.TestCase):
   def test_ja(self) -> None:
     parser = load_default_japanese_parser()
     fp = os.path.join(os.path.dirname(__file__), 'quality', 'ja.tsv')
-    with open(fp) as f:
+    with open(fp, 'r', encoding=sys.getdefaultencoding()) as f:
```

Comment:
Maybe better to use UTF-8? IIUC this will be Shift-JIS if ran on Japanese Windows, but I guess it's not your intention, is it?

Reply:
Thanks for reviewing! Yes, the file is encoded in UTF-8, and I want to open it accordingly. Fixed the code.